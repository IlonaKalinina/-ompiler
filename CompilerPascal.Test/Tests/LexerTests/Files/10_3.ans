1 1 integer 1 1
1 15 separators } }
1 17 integer 2 2
End File