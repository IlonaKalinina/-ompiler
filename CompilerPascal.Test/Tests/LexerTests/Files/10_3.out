1 1 INT 1 1
1 15 SEPARATOR Unidentified }
1 17 INT 2 2
